packages/browsers/src/CLI.ts

```diff
@@ -482,9 +484,16 @@ export function makeProgressCallback(
         },
       );
     }
-    const delta = downloadedBytes - lastDownloadedBytes;
-    lastDownloadedBytes = downloadedBytes;
-    progressBar.tick(delta);
+    if (downloadedBytes >= totalBytes && !isExtracting) {
+      // Transition to the extraction phase
+      progressBar.terminate(); // Clear download progress bar
+      console.log(`Extracting ${browser} ${buildId}...`); // Simple console message for now
```

Comment:
I believe this would report extracting even when no extracting is happening (`options.unpack === false`) and also it  would not cover the waiting for running setup and installing deps (`options.installDeps === true`). 

We probably also should not use console.log directly as the progress bar is only rendering if the [output stream isTTY](https://github.com/visionmedia/node-progress/blob/master/lib/node-progress.js#L121). 

I think using a static message is fine but we should also write it to process.stderr as the progress bar and we can add messages directly to the corresponding code that does extraction/deps/setup conditions on the presence of downloadProgressBar that indicates whether the code is invoked from CLI.